books/scss/detail.scss

```diff
@@ -6,9 +6,26 @@
   padding: 1.5rem;
   width: 56rem;
 
-  .book-box-cover {
-    height: 368px;
-    width: 261px;
+  .book-box-images {
+    position: relative;
+
+    .book-box-cover {
+      height: 368px;
+      width: 261px;
+    } 
+
+    .book-box-badge {
+      height: 7.4rem;
+      position: absolute;
+      right: -1.4rem;
+      top: -1rem;
+      transform: rotate(12deg);
+      -webkit-transform: rotate(12deg);
+      -moz-transform: rotate(12deg);
+      -ms-transform: rotate(12deg);
+      -o-transform: rotate(12deg);
```

Comment:
There's no need to fill all prefixes. In the projects, the autoprefixers will do the job 😄